### ISSUE TITLE
ignoreAtRules don't work

### ISSUE THREAD

**Issue by davidjbradshaw** [stylelint/stylelint#855]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

#803 says it adds the ignoreAtRules option, to allow support for SASS @IF{} @ELSE, however, after upgrading to 4.4.0 and adding this to my config, I still see warnings for this. (I apologize in advance  if this is a case of me being dumb, as the notes on the patch are not clear on how to enable it).

> Which rule, if any, is this issue related to?

```
block-closing-brace-newline-after and at-rule-empty-line-before
```
> What CSS is needed to reproduce this issue?

```css
  @if unit($line-height) == 'px' {
    min-height: $line-height;
  } @else {
    min-height: $line-height * $font-size;
  }
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "extends": "stylelint-config-standard",  
  "rules": {
    "string-quotes": "single",
    "function-url-quotes":none,
    "max-empty-lines": 2,
    "block-closing-brace-newline-after": [
      "always",{
        "ignoreAtRules": ["if", "else"]
      }
    ],
  }
}
```

> Which version of stylelint are you using?

e.g. `4.4.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

e.g. "CLI with `stylelint --config myconfig *.css`"

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

SASS @if{} @else{}

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors you are getting)?

```shell
src/styles/utils/_mixins.scss
9:4	Expected newline after "}" (block-closing-brace-newline-after)
9:5	Expected empty line before at-rule (at-rule-empty-line-before)
```

**Comment by davidjbradshaw** [comment]
Oh just spotted it's in the next release

**Comment by jeddy3** [comment]
> Oh just spotted it's in the next release

No worries. Thanks for the detailed report though :)

You'll find the latest release docs at http://stylelint.io/.

We're very close to releasing `4.5`, so you shouldn't have to wait too long to use that option.
